webpack/app/scss/_basic.scss

```diff
@@ -125,24 +125,45 @@ footer {
   #portalredes-twitter {
     @include sprite($rede-twitter);
   }
+  #portalredes-twitter.inverter {
+    @include sprite($twitter-branco);
+  }
   #portalredes-youtube {
     @include sprite($rede-youtube);
   }
+  #portalredes-youtube.inverter {
+    @include sprite($youtube-branco);
+  }
   #portalredes-flickr {
     @include sprite($rede-flickr);
   }
+  #portalredes-flickr.inverter {
+    @include sprite($flickr-branco);
+  }
   #portalredes-facebook {
     @include sprite($rede-facebook);
   }
+  #portalredes-facebook.inverter {
+    @include sprite($facebook-branco);
+  }
   #portalredes-googleplus {
     @include sprite($rede-googleplus);
   }
+  #portalredes-googleplus.inverter {
+    @include sprite($gplus-branco);
```

Comment:
Ao inves de criar 2 icones, um preto e outro branco, podemos adicionar somente o icone preto, e aplicar o css ``filter: invert(100%);`` para inverter a cor para branco do ícone.

Isso faz com que o arquivo de sprite seja muito menor e possa ser carregado mais rapidamente.

Reply:
Apenas segui o padrão do que já estava desenvolvido, onde alguns ícones tem duas ou mais versões.

Reply:
Mas voce não concorda que fica melhor se for possível ter uma versão do ícone do sprite somente? Qual a dificuldade de alterar (ou remover) o icone e uma linha de CSS?